ocagent.go

```diff
@@ -504,7 +504,7 @@ func (ae *Exporter) uploadTraces(sdl []*trace.SpanData) {
 		ae.senderMu.Lock()
 		err := ae.traceExporter.Send(&agenttracepb.ExportTraceServiceRequest{
 			Spans:    protoSpans,
```

Comment:
I couldn't find an existing test that checks the resource type for exported traces. Any suggestions on how to test this change?

Reply:
you can add resource in addition to span and nodes [here](https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/blob/a8a6f458bbc1d5042322ad1f9b65eeb0b69be9ea/mock_agent_test.go#L128-L129) in  mock_agent_test.go and then modify the test in ocagent_test.go to verify the resources like you have done for metrics.

Reply:
Thanks! I've added a test for checking the resource type on the exported traces.